static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -31,6 +31,25 @@ const enum ErrorHelpType {
   HANDLE_HARD_NAVIGATE_ERROR = 'handle_hard_navigate_error',
 }
 
+const ConditionErrorInfoMap: Record<
+  string,
+  {errorHelpType: ErrorHelpType; projectCheck: boolean}
+> = {
+  ChunkLoadError: {projectCheck: false, errorHelpType: ErrorHelpType.CHUNK_LOAD_ERROR},
+  'window is not defined': {
+    projectCheck: false,
+    errorHelpType: ErrorHelpType.DOCUMENT_OR_WINDOW_OBJECT_ERROR,
+  },
+  'document is not defined': {
+    projectCheck: false,
+    errorHelpType: ErrorHelpType.DOCUMENT_OR_WINDOW_OBJECT_ERROR,
+  },
+  'Invariant: attempted to hard navigate to the same URL': {
+    projectCheck: true,
+    errorHelpType: ErrorHelpType.HANDLE_HARD_NAVIGATE_ERROR,
+  },
+};
```

Comment:
lets make this an array of objects with 3 keys since we're always using it via object.entries

Reply:
`Array<{title: string; errorHelpType: ErrorHelpType; projectCheck: boolean}>`